Makefile

```diff
@@ -52,6 +54,16 @@ lint: bin/golangci-lint ## Run linter.
 lint-fix: bin/golangci-lint ## Fix lint violations.
 	bin/golangci-lint run --fix
 
+lint-markdown: ## Run markdown linter.
+	docker pull -q ${MDLINTER_IMAGE}
+	docker run --rm -v ${PWD}:/workdir ${MDLINTER_IMAGE} \
+		--config testing/markdownlint.yaml -p testing/.markdownlintignore "**/*.md"
+
+lint-markdown-fix: ## Run markdown linter and fix problems automatically.
+	docker pull -q ${MDLINTER_IMAGE}
```

Comment:
`--pull-always` flag for the `run` command should be enough

Reply:
A separate `docker pull` is here to hide the output of the pulling image process (to avoid printing it in the report). 

Maybe another solution exists...